### PR TITLE
Let's Rename Everything 🥬

### PR DESCRIPTION
**Pull request by natemcmaster**
Rename everything to comply with https://letsencrypt.org/trademarks/.

Credit to https://twitter.com/JamesNK/status/1263334400730652677. 

Also, if you don't like puns, blame trademark and copyright law, not me.

### CODE REVIEW

**Comment by Nirmal4G** [comment]
Why not `JustEncrypt`? 🤓🧐

**Comment by natemcmaster** [comment]
> Why not `JustEncrypt`? 🤓🧐

The lettuce pun made me laugh too hard. I was instantly sold.

File: README.md [hunk 1]
```diff
@@ -1,47 +1,43 @@
...
+
 ## Will this work for me?
 
 That depends on [which kind of web server you are using](#web-server-scenarios). This library only works with [Kestrel](https://docs.microsoft.com/aspnet/core/fundamentals/servers/kestrel), which is the default server configuration for ASP.NET Core projects. Other servers, such as IIS and HTTP.sys, are not supported. Furthermore, this only works when Kestrel is the edge server.
 
 Not sure? [Read "Web Server Scenarios" below for more details.](#web-server-scenarios)
 
-Using :cloud: Azure App Services (aka WebApps)? This library isn't for you, but you can still get free HTTPS certificates from Let's Encrypt. See ["Securing An Azure App Service with Let's Encrypt"](https://www.hanselman.com/blog/SecuringAnAzureAppServiceWebsiteUnderSSLInMinutesWithLetsEncrypt.aspx) by Scott Hanselman for more details.
+Using :cloud: Azure App Services (aka WebApps)? This library isn't for you, but you can still get free HTTPS certificates. See ["Securing An Azure App Service with Let's Encrypt"](https://www.hanselman.com/blog/SecuringAnAzureAppServiceWebsiteUnderSSLInMinutesWithLettuceEncrypt.aspx) by Scott Hanselman for more details.
```
Comment (EricStG) [inline README.md:34]: I don't think you wanted to change that link

Comment (natemcmaster) [reply README.md:34]: Good catch!

**Pull request merged**